qa/rpc-tests/test_framework/util.py

```diff
@@ -215,6 +218,9 @@ def start_node(i, dirname, extra_args=None, rpchost=None, timewait=None, binary=
     if binary is None:
         binary = os.getenv("BITCOIND", "bitcoind")
     args = [ binary, "-datadir="+datadir, "-keypool=1", "-discover=0", "-rest" ]
+    if os.getenv("QT", ""):
```

Comment:
I would suggest changing the environment variable name. `QT` seems to be something that could be legitimately used by the toolkit itself, and defined otherwise. Why not `PYTHON_USE_QT` or so? (it's not that descriptive either but stays with the `PYTHON_DEBUG` pattern, and avoids accidental collisions.)